app/controllers/users_controller.rb

```diff
@@ -62,9 +62,36 @@ def filters
     end
   end
 
+  def make_owner_page
+    add_breadcrumb t('users.make_owner'), :make_owner_page_users_path
+    admins = @users.where(role: User::ROLES[1])
+    staff = @users.where(role: User::ROLES[0])
+    @users = staff + admins
+    respond_to { |format| format.html }
+  end
+
+  def make_owner
+    if User.where(email: make_owner_params[:email]).update(role: User::ROLES[2])
+      current_user.update(role: User::ROLES[1])
+      flash[:notice] = t('users.successful_owner_change_message')
```

Comment:
what if for some reason update fails to update? do we still need to show success?

Reply:
resolve